packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -341,6 +340,95 @@ describe('CheckoutModule', () => {
                 // Assert
                 expect(dispatch).toHaveBeenCalledWith(`${VUEX_CHECKOUT_ANALYTICS_MODULE}/updateAutofill`, state, { root: true });
             });
+
+            describe('when the `customer` model is not returned from the api', () => {
+                it('should not error when checking phone number', async () => {
+                    // Arrange
+                    checkoutDelivery.customer = null;
+
+                    // Act
+                    await getCheckout({ commit, state, dispatch }, payload);
+
+                    // Assert
+                    expect(checkoutDelivery.customer).toBe(null);
+                    expect(axios.get).toHaveBeenCalledWith(payload.url, config);
```

Comment:
Do we need this expect and the following expect in each test? Think just the one above is fine?